garena_parser.py

```diff
@@ -0,0 +1,244 @@
+    if team["win"] == True or team["win"] == "Win":
+        winner_team_id = int(team["teamId"])
+
+if winner_team_id == 100:
+    winner = "1"
+elif winner_team_id == 200:
+    winner = "2"
+else:
+    print("No winner team could be found!")
+    exit()
+
+if winner != winner_data:
+    print("Winner team does not match!")
+    exit()
+
+# teamstats = {"team1": {}, "team2": {}}
+
+participantstats = []
+participantids_dict = {
+    0: "blue1",
+    1: "blue2",
+    2: "blue3",
+    3: "blue4",
+    4: "blue5",
+    5: "red1",
+    6: "red2",
+    7: "red3",
+    8: "red4",
+    9: "red5"
+}
+
+rune_keys = ["primary_1", "primary_2", "primary_3", "primary_4", "secondary_1", "secondary_2", "offense", "flex",
+             "defense"]
+
+player_templates = ["", ""]
+teamgold = {"team1": 0, "team2": 0}
+
+for i, participant in enumerate(match_data["participants"]):
+    participantinfo = {
+        "link": participant["summonerName"].split(" ")[1],
+        "champion": champion_ids[participant["championId"]],
+        "kills": str(participant["kills"]),
+        "deaths": str(participant["deaths"]),
+        "assists": str(participant["assists"]),
+        "gold": str(int(participant["goldEarned"])),
+        "cs": str(int(participant["totalMinionsKilled"]) + int(participant["neutralMinionsKilled"])),
+        "visionscore": str(int(participant["visionScore"])),
+        "damagetochamps": str(int(participant["totalDamageDealtToChampions"])),
+        "summonerspell1": spell_ids[participant["spell1Id"]],
+        "summonerspell2": spell_ids[participant["spell2Id"]],
+        "trinket": item_ids[participant["item6"]],
+        "pentakills": str(participant["pentaKills"]),
+        "primary": rune_ids[participant["perks"]["styles"][0]["style"]],
+        "secondary": rune_ids[participant["perks"]["styles"][1]["style"]],
+        "keystone": rune_ids[participant["perks"]["styles"][0]["selections"][0]["perk"]],
+        "primary_1": rune_ids[participant["perks"]["styles"][0]["selections"][0]["perk"]],
+        "primary_2": rune_ids[participant["perks"]["styles"][0]["selections"][1]["perk"]],
+        "primary_3": rune_ids[participant["perks"]["styles"][0]["selections"][2]["perk"]],
+        "primary_4": rune_ids[participant["perks"]["styles"][0]["selections"][3]["perk"]],
+        "secondary_1": rune_ids[participant["perks"]["styles"][1]["selections"][0]["perk"]],
+        "secondary_2": rune_ids[participant["perks"]["styles"][1]["selections"][1]["perk"]],
+        "offense": rune_ids[participant["perks"]["statPerks"]["offense"]],
+        "flex": rune_ids[participant["perks"]["statPerks"]["flex"]],
+        "defense": rune_ids[participant["perks"]["statPerks"]["defense"]]
+        }
+    for x in range(0, 6):
+        item = item_ids.get(participant[f"item{str(x)}"])
+        if item != "" and not item:
+            item = participant[f"item{str(x)}"]
+        participantinfo[f"item{str(x + 1)}"] = item
+    ret = ''
+    runes_ret = ''
```

Comment:
these variable names, `ret` and `runes_ret` are kinda unclear? the only times i'd use `ret` is in a really short function when i just need to return a very generic thing, then `ret` is the returned value